### PR TITLE
added table for skills and made it responsive

### PR DESCRIPTION
**Pull request by tarunktk**
-added table view for different months in reports
-responsive

### CODE REVIEW

**Comment by kendrickcarrolldc** [comment]
everything looks good, and meets the acceptance criteria.

File: client/app/main/main.html [hunk 1]
```diff
@@ -87,3 +87,23 @@ <h1 class="md-toolbar-tools">Awesome Menu</h1>
   </md-sidenav>
   <div flex ui-view layout="column"></div>
 </section>
+<!-- <mdt-table  paginated-rows="{isEnabled: true, rowsPerPageValues: [5,10,20]}"  mdt-row="{
```
Comment (lchapoy) [inline client/app/main/main.html]: Erase comment that you are not going to use

File: client/app/main/skills/skills.html [hunk 2]
```diff
@@ -1,142 +1,90 @@
...
+
+  <md-content class="md-padding" layout="column" layout-gt-xs="row">
+    <div flex layout-gt-xs="column">
+
+      <md-card>
+
+
+        <span class="md-headline">Table</span>
+        <mdt-table layout-gt-xs="column" paginated-rows="{ isEnabled: true, rowsPerPageValues: [5,10,20]}" mdt-row="{
+                                  data: nutritionList,
```
Comment (lchapoy) [inline client/app/main/skills/skills.html]: Change the data name instead nutritionList to be something more related

File: client/app/main/skills/skillscontroller.js [hunk 3]
```diff
@@ -1,49 +1,385 @@
...
+      }, {
+
+        first_name: 'Nitin',
+        last_name: 'kondapalli',
+        client: 'Baylor'
+      }, ]
+
+    }
+    this.updateMonth = function(month) {
+      $scope.nutritionList = reports[month]
```
Comment (lchapoy) [inline client/app/main/skills/skillscontroller.js]: Change name of nutritionList

File: client/app/main/skills/skillscontroller.js [hunk 4]
```diff
@@ -1,49 +1,385 @@
...
+    //   switch (month) {
+    //     case 'January':
+    //       $scope.nutritionList = [{
+    //
+    //         first_name: 'Tarun',
+    //         last_name: 'Kondapalli',
+    //         client: 'Facebook'
+    //       }, {
+    //
```
Comment (lchapoy) [inline client/app/main/skills/skillscontroller.js]: Erase comment that you are not going to use

**Review by lchapoy — CHANGES_REQUESTED** [review]
Clean code

**Comment by lchapoy** [comment]
Some changes requested

**Review by lchapoy — APPROVED** [review]

**Pull request merged**